flask_security/__init__.py

```diff
@@ -1,13 +1,13 @@
 # -*- coding: utf-8 -*-
 """
-    flask_security
```

Comment:
If you want to not get blamed for this, you can add the black commit to [git-blame-ignore-revs](https://github.com/inveniosoftware/flask-security-fork/blob/master/.git-blame-ignore-revs). Up to you.